client/lib/signup/step-actions.js

```diff
@@ -435,7 +435,7 @@ export function createSite( callback, { themeSlugWithRepo }, { site }, reduxStor
 	const data = {
 		blog_name: site,
 		blog_title: '',
-		public: abtest( 'privateByDefault' ) === 'private' ? -1 : 1,
+		public: -1,
```

Comment:
So that the /start/private flow is still private by default

Reply:
Am I right in thinking this would apply to all flows? Is that intended?

Reply:
After starting at `http://calypso.localhost:3000` and being redirected to `http://calypso.localhost:3000/devdocs/start` and signing up I found the resulting site to be private:
<img width="1413" alt="screen shot 2018-11-15 at 11 44 00" src="https://user-images.githubusercontent.com/4335450/48550881-ce2f1f80-e8cb-11e8-86ae-42d355856828.png">

Reply:
It will only apply to flows that use the `site` step, which are `private` and `developer`.

Reply:
Ahh nice, that makes sense! Good to know!